### DISCUSSION TITLE
Dynamic Reading Lists

### DISCUSSION THREAD

**Discussion by RickSWilliams** [gotson/komga#1642] [Ideas]
I would love the ability to create a dynamic Reading List consisting of entire series of various books and presents them in release order. This would be useful in the case of the small universes books where there are multiple titles that intersect. For example I created a reading list that includes all the issues from all the series in the recent Marvel Ultimate Universe books. I’m about 6 months behind starting it, and would like to read in release order. Reading List works great, but I have to manually add each new issue to it in order to keep it current. Being able to add a series as opposed to issues would allow the reading list to continue to build itself in release order as new issues are released or just a quick way to build a full reading list out of books already released. 
Some examples from my own library where I would find this useful.
1. Fables - various series that tie together at various points. There are several points where it is very important to break off the main series and read spinoff titles in order to understand the story.
2. IDW TMNT - One mainline series with a huge amount of spin off mini series that directly tie back to the main series. 
3. Energon  Universe - recent small Image universe which consists of several series of books. 
4. Any other series where you start far into a run and want an easy way to ensure you don’t miss some spin off stories.

**Comment by gotson** [comment]
duplicate of #1579
